src/main/webapp/js/components/entities/entity-edit.jsx

```diff
@@ -6,21 +6,57 @@ import { Form } from 'react-final-form';
 import Alert from 'react-s-alert';
 import { withRouter } from 'react-router-dom';
 
+import Modal from 'react-modal';
 import { getEntityMemberById } from '../../selectors';
 import renderFormField, { validate } from '../../utils/form/form-utils';
-import { updateEntity } from '../../actions/index';
+import { updateEntity, deleteEntity } from '../../actions/index';
+import ConfirmModal from '../comfirm-modal';
+
+Modal.setAppElement(document.getElementById('root'));
```

Comment:
this is already done in the ConfirmModal, so you can remove this, also you can do the same in FieldConfigModal too